### PR TITLE
Porting System.Runtime.WindowsRuntime.Xaml

### PR DESCRIPTION
**Pull request by dsgouda**
@weshaggard

### CODE REVIEW

File: src/System.Runtime.WindowsRuntime.UI.Xaml/src/System.Runtime.WindowsRuntime.UI.Xaml.csproj [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <AssemblyName>System.Runtime.WindowsRuntime.UI.Xaml</AssemblyName>
+    <ProjectGuid>{263DA4F1-C3BC-4B43-98E7-9F38B419A131}</ProjectGuid>
+    <UseECMAKey>true</UseECMAKey>
+    <AssemblyVersion>4.0.1.0</AssemblyVersion>
```
Comment (weshaggard) [inline src/System.Runtime.WindowsRuntime.UI.Xaml/src/System.Runtime.WindowsRuntime.UI.Xaml.csproj]: Please add the following:
```
<PackageTargetFramework Condition="'$(PackageTargetFramework)'==''">dotnet5.3</PackageTargetFramework>
<PackageTargetRuntime>win8</PackageTargetRuntime>
```

cc @ericstj @pallavit

**Comment by dsgouda** [comment]
@dotnet-bot test  Innerloop OpenSUSE13.2 Debug Build and Test

**Comment by dsgouda** [comment]
@dotnet-bot test Innerloop OpenSUSE13.2 Release Build and Test

**Comment by dsgouda** [comment]
@dotnet-bot  test Innerloop OpenSUSE13.2 Debug Build and Test

**Comment by weshaggard** [comment]
@dotnet-bot test Innerloop OpenSUSE13.2 Debug Build and Test

**Comment by weshaggard** [comment]
LGTM

**Pull request merged**